### DISCUSSION TITLE
S3 data connector with MinIO

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#3889] [Help] [answered]
Hi,
I am looking at configuring a data source with MinIO service but was not able to find a way to configure the endpoint url with the S3 connector.
Is there a way to get around this?
Thanks!

**Comment by shogunpurple** [comment, marked as answer]
Hi @micebrain - I just tried this and realised that it's currently not possible. The reason being that we don't have the `endpoint` parameter being passed to the S3 connector. I will add this into the S3 connector, and it will be available in the next release!

Source: https://docs.min.io/docs/how-to-use-aws-sdk-for-javascript-with-minio-server.html

**Reply by micebrain** [reply]
Thanks @shogunpurple ! Really appreciate the quick response :)
